solutions/rhelai_vpc/variables.tf

```diff
@@ -40,10 +40,38 @@ variable "zone" {
   type        = string
 }
 
+variable "has_existing_vpc" {
```

Comment:
Can we change these "has_" variables to "use_"? We have used the "use_" prefix for these variables in our other DAs so it would be consistent.